lib/meta-boxes.php

```diff
@@ -1,12 +1,11 @@
 <?php
-
 /* Get post objects for select field options */
 function get_post_objects( $query_args ) {
   $args = wp_parse_args( $query_args, array(
-      'posts_per_page' => -1,
-    ) );
+    'post_type' => 'post',
```

Comment:
yeah, the -1 is missing, without it doesn't work as it should
i think `'posts_per_page' => -1` should be in the boilerplate